examples/nodejs-simple/Dockerfile

```diff
@@ -14,17 +14,14 @@
 
 FROM debian:buster
 RUN useradd -u 1000 -m server
-RUN apt-get update && apt-get install -y curl  && apt-get clean
-RUN curl -sL https://deb.nodesource.com/setup_12.x | bash - && \
-    apt-get install -y nodejs
 
-WORKDIR /home/server
+COPY ./examples/nodejs-simple /home/server/examples/nodejs-simple
+WORKDIR /home/server/examples/nodejs-simple
```

Comment:
Minor but if this was moved above the line above could the destination path be simplified in the `COPY` to a relative path?